pkg/printers/internalversion/describe.go

```diff
@@ -624,6 +624,7 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 		w := NewPrefixWriter(out)
 		w.Write(LEVEL_0, "Name:\t%s\n", pod.Name)
 		w.Write(LEVEL_0, "Namespace:\t%s\n", pod.Namespace)
+		w.Write(LEVEL_0, "Priority:\t%d\n", pod.Spec.Priority)
```

Comment:
I don't know how much it is relevant to print the priority because it is still in alpha state.

Have following questions:
- Do we need to print this priority even if the feature is not enabled?
- Even if we decide to print which one makes sense to print? `PriorityClassName` or `Priority`?

@bsalamat any comments.?

Reply:
Thanks @wackxu for this PR. IMO it is premature to print such alpha `Priority`.

> Even if we decide to print which one makes sense to print? `PriorityClassName` or `Priority`

The admission controller populates `Priority` from `PriorityClassName`. But for end-users, I think `Priority` is more intuitive.

Reply:
I think we need to wait priority upgraded to beta :).

Reply:
Priority is an alpha feature, but the fields in PodSpec are not alpha. I don't see any serious problem with showing priority while it is in alpha, but I think we should show both `PriorityClassName` and `Priority`.

Reply:
@bsalamat I agree with the first part.We need print `Priority` when using the feature of pod priority though it is in alpha state. Users concerns more about `Priority` and the PriorityClass may be deleted and re-created with the same name but different priority and it may confuse users when print the same priorityClassName but different priority.

Reply:
I still think both of them are useful for a couple of reasons:
1. The case that you described (a priority class is deleted and added with a different value) will not be resolved by not showing `PriorityClassName`. Imagine this scenario: user creates a pod with "high-priority" class name and the pod is resolved to 1000. Admin changes the value of "high-priority" class from 1000 to 2000. Now, user looks at his pod finds his pod to have 1000 value. He looks at the yaml/JSON/template that he created the pod from and sees that the priority class name was "high-priority". He finds this inconsistent and thinks that there was a bug in the system. If we show "PriorityClassName" in describe, at least the user will know that the priority class of the pod is the same as what he put in his config.

2. It is useful in most cases to know what is the priority class name of the pod.

Reply:
@bsalamat SGTM, addressed.